index.html

```diff
@@ -1,41 +1,60 @@
 ---
 layout: default
 ---
+
 <section class="one">
   <div class="main-image-wrapper">
-    <img src="/images/OutlineHome.png" id="hoverme" UX Computer Setup class="main-image">
-    
+    <img
+      src="/images/OutlineHome.png"
+      id="hoverme"
+      UX
+      Computer
+      Setup
+      class="main-image"
+    />
   </div>
- 
- <h4 class="introduction-text"> 
-  <br>
-  I'm thrilled to start my <span style="color:#ffa700;font-weight: bolder;">UX Apprenticeship</span> and share my journey on this blog. It will be a glimpse into my learning and progress in <span style="color:#ffa700;font-weight: bolder;">UX design</span>. 
-  I'll document challenges and insights, covering all aspects from user research to interaction design. 
-  Follow my adventure as I grow into a skilled UX professional in the dynamic digital design sphere.</h4>
 
-  
+  <h4 class="introduction-text">
+    <br />
```

Comment:
Try to avoid using the <br/> tag. Refactor this to get at the desired positioning with something like padding or margin.